prow/config.yaml

```diff
@@ -29,7 +29,7 @@ tide:
         - do-not-merge/work-in-progress
 
 plank:
-  job_url_prefix: https://gubernator.k8s.io/build/
+  job_url_prefix: https://prow.gflocks.com/view/gcs/
```

Comment:
We should probably do this in a follow up so that we can make sure spyglass is WAI before making it the default result viewer.

Reply:
good point, dropped this change